docs/customization/export-templates.md

```diff
@@ -32,6 +32,8 @@ If you need to use the config context data in an export template, you'll should
 {% endfor %}
 ```
 
+To see all the attributes of a given object, you can use the [Netbox Shell](../administration/netbox-shell.md). It supports autocompletion and allows one to see all of the methods and attributes a given object type has. All of them can be called from within a Jinja template. Using queries from one object to another, one can navigate pretty much the entire Netbox object model. For instance, from an export template for sites, one can get the devices that are in that site, the circuits that are connected to those devices, the providers that serve those circuits... etc, so an export template is not limited to just the model that it's being called from. In fact, the same result can be achieved in different ways, depending on which model you start from.
```

Comment:
This should be reduced to a "tip" admonition as the topic strays from export templates. Just point out that object attributes can be explored using `nbshell`.